docker/Dockerfile-api

```diff
@@ -26,8 +26,16 @@ RUN pip3 install . --no-deps
 # Use the system CA bundle for the requests library
 ENV REQUESTS_CA_BUNDLE=/etc/pki/tls/certs/ca-bundle.crt
 
+# Temp dir used to collate the metrics between different threads by
+# the Prometheus Flask Exporter
+RUN mkdir /tmp/wsgi
+
 # Disable gitpython check for the git executable, cachito-api doesn't use git
 ENV GIT_PYTHON_REFRESH=quiet
 
+# Environment variables used by the Prometheus Flask exporter.
+ENV PROMETHEUS_MULTIPROC_DIR /tmp/wsgi
+ENV DEBUG_METRICS true
```

Comment:
Do we want debug by default? This Dockerfile is also used for production images, not just for testing

Reply:
... turned off.